mod/groups/views/default/groups/listing/discussion.php

```diff
@@ -0,0 +1,5 @@
+<?php
```

Comment:
but in the resource view you check if the selected tab view exists. What exceptions are you revering to?